BnbnavNetClient.Linux/Program.cs

```diff
@@ -1,33 +1,28 @@
-﻿// See https://aka.ms/new-console-template for more information
-
-using Avalonia;
+﻿using Avalonia;
 using Avalonia.ReactiveUI;
-using Avalonia.Svg.Skia;
 using BnbnavNetClient.I18Next;
 using BnbnavNetClient.Linux.TextToSpeech;
 using BnbnavNetClient.Services.TextToSpeech;
 using BnbnavNetClient.Settings;
 
 namespace BnbnavNetClient.Linux;
 
-internal class Program
+internal static class Program
 {
     public static void Main(string[] args)
     {
         BuildAvaloniaApp().StartWithClassicDesktopLifetime(args);
     }
 
-    static AppBuilder BuildAvaloniaApp()
+    public static AppBuilder BuildAvaloniaApp()
     {
-        GC.KeepAlive(typeof(SvgImageExtension).Assembly);
-        GC.KeepAlive(typeof(Avalonia.Svg.Skia.Svg).Assembly);
```

Comment:
Please re-add these for Rider Avalonia plugin support

Reply:
It seems this is only really necessary if you're using them from axaml, to make the designer happy. (https://github.com/AvaloniaUI/AvaloniaVS/issues/250)